src/container/srv_container.c

```diff
@@ -5529,6 +5528,20 @@ ds_cont_op_handler(crt_rpc_t *rpc, int cont_proto_ver)
 	struct cont_svc			*svc;
 	int				 rc;
 
+	/*
```

Comment:
nit: is this comment style consistent with guidelines?

Reply:
Ah, I forgot the line of `*`s along the side. Will update if there's anything else to change.

Reply:
Fixed